docs/source/about/team.rst

```diff
@@ -50,34 +52,75 @@ Core Developers
 
    * - Name
      - GitHub ID
+     - Current affiliation of contribution
+     - Affiliation of primary role
    * - Aaron Bostrom
      - :user:`abostrom`
+     - University of East Anglia
+     - University of East Anglia
    * - Anthony Bagnall
      - :user:`TonyBagnall`
+     - University of East Anglia
+     - University of East Anglia
    * - Chris Holder
      - :user:`chrisholder`
-   * - Guzal Bulatova
-     - :user:`GuzalBulatova`
+     - University of East Anglia
+     - University of East Anglia
    * - Franz Kiraly
      - :user:`fkiraly`
+     - Shell Research; University College London (honorary)
+     - Shell Research
    * - Freddy A Boulton
      - :user:`freddyaboulton`
+     - Alteryx
+     - Alteryx
    * - George Oastler
      - :user:`goastler`
-   * - Markus Löning
-     - :user:`mloning`
+     - University of East Anglia
+     - University of East Anglia
+   * - Guzal Bulatova
+     - :user:`GuzalBulatova`
+     - Individual
+     - Eneryield
    * - James Large
      - :user:`james-large`
+     - Individual
+     - Alfa iQ
+   * - Jason Lines
+     - :user:`jasonlines`
+     - University of East Anglia
+     - University of East Anglia
+   * - Markus Löning
+     - :user:`mloning`
+     - Individual; Shell Research
+     - Freelancer; Shell Research (contracting)
    * - Lovkush Agarwal
      - :user:`lovkush-a`
+     - Individual
+     - Freelancer; Shell Research (contracting)
    * - Martin Walter
      - :user:`aiwalter`
+     - Mercedes-Benz
+     - Mercedes-Benz
```

Comment:
```suggestion
     - Individual
     - Individual
```

Reply:
Ok, I've changed this - is it ok (since former) or do I need to add more details?